imports/ui/components/inactiveClusters/component.html

```diff
@@ -25,13 +25,15 @@
             <tr>
               <th>Cluster</th>
               <th>Last active</th>
+              <th>Action</th>
             </tr>
           </thead>
           <tbody>
             {{#each cluster in zombieClusters}}
             <tr>
               <td><a href="{{pathFor 'cluster.tab' id=cluster.cluster_id tabId='resources'}}">{{getClusterName cluster}}</a></td>
               <td>{{moment cluster.updated}}</td>
+              <td><button class="delete" cluster_id="{{cluster.cluster_id}}">Remove</button></td>
```

Comment:
can you add the `btn` and `btn-primary` classes here?  That will pull in the bootstrap button classes that we use elsewhere in razeedash

Reply:
Added in #7b65142